Terms of Service

JamesVorder opened the pull request:
Please have a look. I think it would be a good idea to adopt the Network Commons License for the same reasons we talked about briefly during last tech meetup. There are possibly some specifics around gathering responses that we should talk about, too.

JamesVorder committed:
Add policy for ToS

JamesVorder committed:
Added NCL as tos.md

Commit:
Link to ToS from policy.

JamesVorder committed:
Move tos to ncl

Commit:
Remove document's version number.

cure commented on tos.md:
```diff
@@ -0,0 +1,84 @@
+allow other network participants to install devices in their facilities, but this permission is always
+discretionary and can be revoked at any time.
+- Commercial access to the network must extend and share the network.
+- Data from the network is not logged or stored.
```
we probably need some carveout here for operation of the network, we may occasionally need to log some data to track down a service problem, say.

JamesVorder replied:
Agreed. Good catch

cure commented on ncl.md:
```diff
@@ -1,4 +1,4 @@
-# Network Commons License (v1.0.4)
+# Network Commons License
```
I suggest we keep the version number if we're not changing anything. It will be easier to adopt future versions if we know where we start.

cure commented on tos.md:
```diff
@@ -0,0 +1,84 @@
+- In order to maximize connectivity and network growth, network constituents are encouraged to
+allow other network participants to install devices in their facilities, but this permission is always
+discretionary and can be revoked at any time.
+- Commercial access to the network must extend and share the network.
```
I'm not sure how this is different from the point above, "Network participants agree to extend the network on the same terms and conditions that they ￼receive it"

JamesVorder replied:
Agreed. This has been deleted.

cure commented:
All my comments appear to be about things that were added since v1.0.2 (https://www.nycmesh.net/ncl.pdf) which is the one that NYC mesh seems to use. Curious.

I also found the license used by guifi.net, which seems a bit more expansive: http://guifi.net/en/FONNC

JamesVorder committed:
Re-add version number.

JamesVorder committed:
Add carveout for network maint.

JamesVorder committed:
Guifi wording in 4th summary bullet.

JamesVorder committed:
Add 'MM Role' clause from Guifi

JamesVorder commented:
@cure Thanks for sharing the link to the FONN license that Guifi is using. I added a section to our own TOS based on theirs... "About The Role of Mass Mesh." I think this is pretty important for us to clarify. Can you have a look and tell me if I missed anything else there? I basically paraphrased what they had in the FONN, and added some basic direction about how we relate to neighborhood networks.

Commit:
Clarify opening section

- Pulled "About MM" section up to near the top, with other summary info. 
- Eliminate redundant heading
- Make it clear what our goal is by adopting this document

Commit:
Remove fluffy bullet point from "Terms of Use"

Commit:
Security section

- Bold the Internet non-guarantee for clarity
- Move Security to its own section (was typo b4)

Commit:
Remove typo in 4th bullet
